service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/GatekeeperEmailContentProvider.java

```diff
@@ -16,7 +16,14 @@ public class GatekeeperEmailContentProvider extends SharedNotifyContentProvider
     public NotifyGatekeeperTemplate buildGatekeeperNotification(CaseData caseData) {
         NotifyGatekeeperTemplate template = buildNotifyTemplate(NotifyGatekeeperTemplate.builder().build(), caseData);
 
-        template.setLocalAuthority(config.getLocalAuthorityName(caseData.getCaseLocalAuthority()));
+        // Check if local authority has a value otherwise use 3rd party
+        if (caseData.getCaseLocalAuthority() != null && !caseData.getCaseLocalAuthority().isEmpty()) {
```

Comment:
ObjectUtils has a `isEmpty(variable)` that combines the null check + the length check all in one which could be used here!